Fixing typos in the explanation of the role of activeDeadlineSeconds (#29341)
kubernetes/website: content/en/docs/concepts/workloads/pods/init-containers.md

Use `activeDeadlineSeconds` on the Pod to prevent init containers from failing forever.
The active deadline includes init containers.

> However it is recommended to use `activeDeadlineSeconds` if user deploy their application

Reviewer: this seems the correct explanation for `activeDeadlineSeconds`. no change needed here.

> Do not use `activeDeadlineSeconds` if you deploy your application as a Job, because `activeDeadlineSeconds` has an effect even after init containers have finished. Setting `activeDeadlineSeconds` would terminate correctly  running Pods.

Reviewer: it is recommended to use `activeDeadlineSeconds` if user deploy their application.

Use `activeDeadlineSeconds` on the Pod to prevent init containers from failing forever.
The active deadline includes init containers.

> However it is NOT recommended to use `activeDeadlineSeconds` if the user deploys their application
> as a Job, because `activeDeadlineSeconds` has an effect even after init containers have finished.
> The Pod which is already running correctly would be killed by `activeDeadlineSeconds` if you set it.

Reviewer 2: ```suggestion
Do not use `activeDeadlineSeconds` if you deploy your application as a Job, because `activeDeadlineSeconds` has an effect even after init containers have finished. Setting `activeDeadlineSeconds` would terminate correctly  running Pods.
```

Some optional related cleanup in this paragraph. LGTM though.
